src/api/deleteOrderRequest.ts

```diff
@@ -2,28 +2,12 @@
 import { AccountDataType, AccountConfigType } from '../types'
 
 // Import debug console log
-import { debug } from '../utils'
+import { debug, getFetchRequestOptions } from '../utils'
 
 export function deleteOrderRequest(orderId: String, accountData: AccountDataType, accountConfig: AccountConfigType): Promise<void> {
   return new Promise((resolve, reject) => {
 
-    const requestOptions: {
-      method?: string,
-      body?: string,
-      headers: {
-        [key: string]: string,
-      },
-      credentials: 'include',
-      referer: string,
-    } = {
-      method: 'DELETE',
-      headers: {
-        'Content-Type': 'application/json;charset=UTF-8',
-      },
-      body: '',
-      credentials: 'include',
-      referer: 'https://trader.degiro.nl/trader/',
-    }
+    const requestOptions = getFetchRequestOptions(accountConfig.data.sessionId, 'DELETE', '', 'application/json;charset=UTF-8')
```

Comment:
Indeed, I didn't try to address issues of logic what is required where, but it seems like it would be optional indeed. Related is that I also see a lot of doubling of jsessionid in the parameters and headers, not sure if both are required in many cases.